club/parts/footer.html

```diff
@@ -1,11 +1,29 @@
-<!-- wp:group {"layout":{"inherit":true}} -->
-<div class="wp-block-group">
-	<!-- wp:group {"style":{"spacing":{"padding":{"top":"80px","bottom":"30px"}}}} -->
-	<div class="wp-block-group" style="padding-top:80px;padding-bottom:30px">
-		<!-- wp:paragraph {"align":"center"} -->
-		<p class="has-text-align-center">Proudly Powered by <a href="https://wordpress.org" rel="nofollow">WordPress</a></p>
-		<!-- /wp:paragraph -->
-	</div>
+<!-- wp:group -->
+<div class="wp-block-group"><!-- wp:group {"layout":{"type":"flex","flexWrap":"nowrap","justifyContent":"space-between"}} -->
+	<div class="wp-block-group"><!-- wp:group {"style":{"spacing":{"blockGap":"0.5rem"}},"layout":{"type":"flex","orientation":"vertical"}} -->
+	<div class="wp-block-group"><!-- wp:site-title {"level":2,"style":{"typography":{"lineHeight":"1.6"}},"fontSize":"default-title"} /-->
+	
+	<!-- wp:paragraph {"fontSize":"default-title"} -->
+	<p class="has-default-title-font-size">Club 6 West 25th St, NY 10010</p>
+	<!-- /wp:paragraph --></div>
 	<!-- /wp:group -->
-</div>
-<!-- /wp:group -->
\ No newline at end of file
+	
+	<!-- wp:group {"style":{"spacing":{"blockGap":"0.5rem"}},"layout":{"type":"flex","orientation":"vertical","justifyContent":"right"}} -->
+	<div class="wp-block-group"><!-- wp:paragraph {"align":"right","fontSize":"default-title"} -->
+	<p class="has-text-align-right has-default-title-font-size">info@cl.ub</p>
+	<!-- /wp:paragraph -->
+	
+	<!-- wp:paragraph {"align":"right","fontSize":"default-title"} -->
+	<p class="has-text-align-right has-default-title-font-size">+12345678910</p>
+	<!-- /wp:paragraph --></div>
+	<!-- /wp:group --></div>
+	<!-- /wp:group -->
+	
+	<!-- wp:paragraph -->
+	<p>Powered by WordPress</p>
+	<!-- /wp:paragraph -->
+	
+	<!-- wp:spacer {"height":"1rem"} -->
+	<div style="height:1rem" aria-hidden="true" class="wp-block-spacer"></div>
+	<!-- /wp:spacer --></div>
+	<!-- /wp:group -->
```

Comment:
The contents of this file should probably be a pattern, as we need to escape and translate all the text content.

Reply:
I think that each user will fill the footer with their own data and not translate this fake address data included in the theme, so adding the translatable strings seems pointless in this case,